infrastructure/src/main/java/org/corfudb/infrastructure/AbstractServer.java

```diff
@@ -6,11 +6,17 @@
 import lombok.extern.slf4j.Slf4j;
 import org.corfudb.protocols.wireprotocol.CorfuMsg;
 
+import java.time.Duration;
+import java.util.concurrent.TimeUnit;
+
 /**
  * Created by mwei on 12/4/15.
  */
 @Slf4j
 public abstract class AbstractServer {
+    public static final long NON_LOG_ADDR_MAGIC = -1L;
+    public static final Duration SMALL_INTERVAL = Duration.ofMillis(60_000);
```

Comment:
added a comment.

I agree, down the road, we want to make these configurable, perhaps by CorfuServer. For now, it's better to concentrate and name these duration parameters than no to.